openapi.yaml

```
@@ -17377,6 +17378,7 @@ components:
           x-linode-cli-display: 6
         type:
           type: string
+          x-linode-filterable: true
           description: >
             How the Image was created. Manual Images can be created at any time.
```

To be consistent with the way manual is described above.

```suggestion
            How the Image was created. "Manual" Images can be created at any time.
```